index.js

```diff
@@ -1437,6 +1437,22 @@
     }
   };
 
+  //# not :: Boolean -> Boolean
+  //.
+  //. Takes a Boolean and returns the negation of that value
+  //. (`false` for `true`; `true` for `false`).
+  //.
+  //. ```javascript
+  //. > S.not(true)
+  //. false
+  //.
+  //. > S.not(false)
+  //. true
+  //. ```
+  S.not = def('not', [Boolean], function(x) {
+    return !x.valueOf();
+  });
```

Comment:
Oh, you're right! It's good that we have tests to prevent us from forgetting JavaScript's more questionable behaviour!